### PR TITLE
Ceph WFFC lane set up

### PR DESCRIPTION
**Pull request by akalenyu**
**What this PR does / why we need it**:
We aim to support WFFC storage classes seamlessly so a periodic run from time to time may
increase our confidence in such setup

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
kubevirtci now gives us both an Immediate & WFFC storage class https://github.com/kubevirt/kubevirtci/pull/967

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: automation/ceph-wffc.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2022 Red Hat, Inc.
+#
+
+set -ex
+export TARGET=k8s-1.24
```
Comment (awels) [inline automation/ceph-wffc.sh:21]: Any reason not to use 1.25?

**Review by awels — COMMENTED** [review]
/approve
looks good to me, just one random question.

File: automation/ceph-wffc.sh (hunk 1 above)
Comment (akalenyu) [reply automation/ceph-wffc.sh:21]: Not really, just sticking close to the current ceph script

File: cluster-sync/k8s-1.23/provider.sh
Comment (oshoval) [inline cluster-sync/k8s-1.23/provider.sh:1]: 1.23 is not supported anymore

Comment (akalenyu) [reply cluster-sync/k8s-1.23/provider.sh:1]: removed together with 1.22

Comment (oshoval) [reply cluster-sync/k8s-1.23/provider.sh:1]: lgtm in this aspect, thanks

**Comment by awels** [comment]
Fixed it again, even though we told it to exclude /cluster-up it is still checking.

**Comment by akalenyu** [comment]
> Fixed it again, even though we told it to exclude /cluster-up it is still checking.

Also lost your lgtm, basically didn't change anything other than removing some outdated sync providers

**Pull request merged**